src/NCronJob/JobExecutionContext.cs

```diff
@@ -26,9 +26,6 @@ internal sealed record JobExecutionContext : IJobExecutionContext
     /// <summary>The Job Run instance.</summary>
     internal JobRun JobRun { get; }
 
-    /// <summary>The Type that represents the Job</summary>
-    internal Type JobType => JobRun.JobDefinition.Type;
```

Comment:
Dropped as it only acts as a passthrough and was obscuring my impact analysis of `JobDefinition.Type` usage